### DISCUSSION TITLE
Create and apply KTX2 texture

### DISCUSSION THREAD

**Discussion by gjhatzis** [google/model-viewer#4002] [Q&A]
Greatings, 

When I am trying to load a png texture to my model (image1), everything appears correctly, but when I try to use a KTX2 file the baseColorTexture appears to be null.

Is it possible to do it?Am I doing something wrong?

Thanks in advance,
George

image1
![image](https://user-images.githubusercontent.com/62299083/207085382-ef6e5718-acb0-40a0-a708-aad192055973.png)

**Comment by elalish** [comment]
We don't support this yet, but it should just be a matter of hooking up the `KTX2Loader` to `createTexture`. A PR would be most welcome!

**Reply by gjhatzis** [reply]
Thank you for you response! I'll try to implement it!

**Reply by danielradst** [reply]
@gjhatzis @elalish any updates on this? I'm trying to create and apply ktx2 textures as well however failing to do so.

**Reply by elalish** [reply]
Sorry, this still hasn't made it up our priority list - pretty short-staffed. I'd be happy to review a PR though!

**Reply by danielradst** [reply]
> Sorry, this still hasn't made it up our priority list - pretty short-staffed. I'd be happy to review a PR though!

Any chance you got time now? 🥇

**Reply by danielradst** [reply]
Hey @diegoteran @samaneh-kazemi 
 
 I see that @elalish unfortunately is not part of the team anymore. Any chance you can check this out? Thanks in advance!

**Comment by diegoteran** [comment]
@MyrqvistDouglas submitted #5141 , thank you
